### PR TITLE
v1.0.0-rc2 release

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.13.90
+1.0.90-rc2
```
Comment (joestringer) [inline VERSION]: This version number will be >`1.0`, which means that typical upgrade scripts would attempt to change to this RC in favour of the final `1.0` release. Usually when the `.90` style is used, the previous number is picked. For instance, `0.13.90`.

Comment (tgraf) [reply VERSION]: An argument can be made for either model as we want the current rcs to be upgraded to the latest version as well. It doesn't really matter that match as we are not even shipping as packages. I'm fine either way though.

Comment (tgraf) [reply VERSION]: My thought here was that we would *not* bump the VERSION in master until we actually release `1.1`so as we put stable releases out `1.0.1`, `1.0.2` they would still always be upgraded to the latest master.

Comment (joestringer) [reply VERSION]: I would have naturally expected one of the following paths:

```
0.12
0.13.90 (master was this prior to RC release)
0.13.91 (use for RC1)
0.13.92 (use for RC2)
0.13.93 (use for RC3)
1.0.0 (final release, version is strictly larger than RCs)
1.0.1 (bugfix update)
1.0.90 (dev version on master for 1.1 release)
1.1.0 (eventual 1.1 release)
```
or
```
0.12
0.13.90 (master prior to RC)
1.0.0-rc2 (Now)
1.0.0-rc3 (use for next rc)
1.0.0 (final release, version is rcs with the rc bit stripped)
1.0.1 (bugfix update)
1.0.90 (dev version for master for 1.1 release)
1.1.0 (eventual 1.1 release)
```

However, it seems like maybe in general Cilium has typically used the `0.13.90` version for pre-`0.13` development? In that case this is consistent, which is arguably the more important point.

**Review by aalemayhu — COMMENTED** [review]
Is this still relevant?

**Comment by tgraf** [comment]
test-me-please

**Pull request merged**